src/upgrade.js

```diff
@@ -116,13 +118,12 @@ const applyPatch = async (
 /**
  * Upgrade application to a new version of Apollos.
  */
-async function upgrade({ from: fromVersion, to: toVersion, platform, projectName, packageName }) {
+async function upgrade({ from: fromVersion, platform, projectName, packageName }) {
   const tmpPatchFile = `tmp-upgrade-apollos-${platform}.patch`;
-  const projectDir = '.';
+  const projectDir = '.'; 
 
-  // todo, figure this out automatically
-  const currentVersion = fromVersion || '0.8.0-alpha.4';
-  const newVersion = toVersion || '0.8.1';
+  const currentVersion = semver.ltr(mostRecentVersion, fromVersion) ? fromVersion : mostRecentVersion;
```

Comment:
I think this always needs to be either `fromVersion` from the user, or we need to read the version from their local repo / package.json.

Reply:
I think that’s a great idea. So instead of requiring them to pass a CLI command we just pull from their current directory? Would make even more since if core had versions file

Reply:
I'm working on this @vinnyjth, but I did update the other issues for now.